news/2024/2024-04-09-skinning-contest-unchained.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "Skinning Contest: Unchained"
+date: 2024-04-09 15:00:00 +0000
+---
+
+Showcase your unshackled creativity to its fullest by participating in the newest skinning contest!
+
+![](/wiki/shared/news/2024-04-09-skinning-contest-unchained/banner.jpg)
+
+The skinship team has thrown all caution to the wind and unlocked the ultimate skinning contest theme.... which is that there isn't one. Yes, you heard that right!
+
+With a throwback to the [1st skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement) and a new contest schedule that gives us 2 contests each year around spring and fall time, the skinning community will once again be able to unleash their inner artist with no restrictions as **there is no theme restriction for this contest.** That means that you can make a skin centred around anime girls, video games that heavily involve anime girls, online personalities that are themselves anime girls, or perhaps a skin that doesn't include any anime at all (don't worry, we won't judge you).
```

Comment:
> with no restrictions as **there is no theme restriction for this contest.**

this sounds dumb, saying way too similar things right after each other

Reply:
423c6b51ef213b76afb6526f74345a676c68cfd0 sorted